app/templates/components/public/session-item.hbs

```diff
@@ -44,7 +44,7 @@
   <div class="content">
     {{#each session.speakers as |speaker|}}
       <div class="ui divider"></div>
-      <img alt="speaker" class="ui tiny avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
+      <img alt="speaker" class="ui avatar small image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
```

Comment:
There is no need to change class, change iconImage to larger image variant

Reply:
ok

Reply:
Use thumbnailImageUrl instead